db/fetch_db.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../lib/poll'
+require_relative '../lib/vote'
+require 'pg'
+
+=begin
+  データベースにアクセスして投票や票の
+  情報を取得する、また票のカウントを
+  行うクラス
+=end
+
+class Fetch_DB
+  attr_reader :host, :database, :user, :connection, :polls
+
+  #初期設定（データベースの接続や票の情報の取得）
+  def initialize(host, database, user, password)
+    @host = host
+    @database = database
+    @user = user
+    @password = password
+    @connection = connect(host, database, user, password)
+    if @connection == nil #データベースの接続のチェック
+      @polls = nil
+    else
+      @polls = get_polls()
+    end
+  end
+
+  #データベースに接続し、接続情報を返すメソッド
+  def connect(host, database, user, password)
+    connection = PG::Connection.new(:host => host, :user => user, :dbname => database, :port => '5432', :password => password)
+    connection
+  rescue PG::ConnectionBad #データベースに接続できなかった場合
+    puts "データベースにアクセス出来ませんでした"
+    nil #nilを返す
+  end
+
+  #データベースのテーブルを作成するメソッド
+  def create_tables()
+    sql = File.open('db/create_tables.sql', 'rb') { |file| file.read } #SQLを読み込む
```

Comment:
[nit] 細かいですが、`File.read('db/create_tables.sql')` だけで読み込めます。

```suggestion
    sql = File.read('db/create_tables.sql') #SQLを読み込む
```